src/finn/util/platforms.py

```diff
@@ -461,11 +461,46 @@ def compute_resources(self):
         ]
 
 
+class Alveo_NxU55C_Platform(Platform):
```

Comment:
How did you determine these settings for U55C?

Reply:
According to our knowledge, the U55C is identical to the U280 cards (based on https://github.com/erinadreno/list_of_Xilinx_FPGAs#Alveo-and-Kria) in regards to the PL.
Therefore, this calculation is based on the calculations for the U280 card that is already included.

Reply:
Thanks, could you please update the metrics according to [UG1102](https://docs.xilinx.com/r/en-US/ug1120-alveo-platforms/U55C-Gen3x16-XDMA-base_3-Platform)? 
E.g. the number of slr (`nslr`) for the U55C is 3 and not 2 and the ethernet connection is on SLR1. In the documentation from the link above you can also find the resources per SLR that are not completely identical with the U280. I would also suggest, that the lines that are part of "observed from Vivado" are taking out, unless you have experimental results for that.

Reply:
Everything should be fixed now based on the UG1102 document.